Test coverage broken after running build

After building generates an index.d.ts, there is an error when running `npm run test-unit`

```
ERROR in [default] C:\www\seng-boilerplate\index.d.ts:1:35
Cannot find module './lib/Example'.

ERROR in [default] C:\www\seng-boilerplate\index.d.ts:2:36
Cannot find module './lib/IExample'.
```

After this our coverage is no longer generated by remap-istanbul

`13 07 2016 06:44:41.318:WARN [reporter.remap-istanbul]: Could not find any specified files, exiting without doing anything.`

--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
     "check-coverage": "istanbul check-coverage --statement 1 --branches 1 --functions 1 --lines 1",
     "validate-webpack": "webpack-validator config/webpack.config.js",
     "lint": "tslint -c tslint.json src/**/*.ts",
-    "clean": "rimraf dist lib index.js index.d.ts coverage 'seng-*.@(zip|tar.gz)'",
+    "clean": "rimraf dist lib index.js index.d.ts coverage \"seng-*.@(zip|tar.gz)\"",
     "compile": "npm-run-all typescript-npm typescript-system typescript-es6 webpack-dist",
     "typescript-npm": "tsc -p ./ -d",
     "typescript-system": "tsc -p ./ -m system --outFile ./dist/seng-boilerplate-system.js",

